apps/server/server.cpp

```diff
@@ -941,6 +941,21 @@ void invoke_sdk_api(payload::ClientRequest buff_recv) {
         break;
     }
 
+    case GET_INI_ARRAY: {
+        std::string imagerType = buff_recv.func_strings_param(0);
+        int mode = buff_recv.func_int32_param(0);
+        std::string iniStr;
+
+        aditof::Status status =
+            camDepthSensor->getIniParamsArrayForMode(imagerType, mode, iniStr);
```

Comment:
Can we remove imagerType from this api? The sensor should know which data to sens.